pgx/_src/gardner_chess_utils.py

```diff
@@ -32,18 +33,18 @@
         c = c + dc[plane - 9]
         if r < 0 or r >= 5 or c < 0 or c >= 5:
             continue
-        to = jnp.int32(c * 5 + r)
-        TO_MAP = TO_MAP.at[from_, plane].set(to)
-        PLANE_MAP = PLANE_MAP.at[from_, to].set(jnp.int32(plane))
+        to = c * 5 + r
+        TO_MAP[from_, plane] = to
+        PLANE_MAP[from_, to] = plane
 
 
-CAN_MOVE = -jnp.ones((7, 25, 16), jnp.int32)
+CAN_MOVE = -np.ones((7, 25, 16), np.int32)
 # usage: CAN_MOVE[piece, from_x, from_y]
 # CAN_MOVE[0, :, :] are all -1
 # Note that the board is not symmetric about the center (different from shogi)
 # You can imagine that the viewpoint is always from the white side.
 # Except PAWN, the moves are symmetric about the center.
-# We define PAWN as a piece that can move up, down, and diagonally, and filter it according to the turn.
+# We define PAWN as a piece that can move up and diagonally.
```

Comment:
I'm not sure why, but it seems that for gardner chess, this comment was previously untrue, as the following condition ([line 56](https://github.com/Akulen/pgx/blob/94667e7d6f6380059443bcb5d9e8be2f08c4ae03/pgx/_src/gardner_chess_utils.py#L56)) is
```python3
if r1 - r0 == 1 and np.abs(c1 - c0) <= 1:
```
instead of
```python3
if np.abs(r1 - r0) == 1 and np.abs(c1 - c0) <= 1:
```

Reply:
This will be investigated in #1174 
Thank you for your comment!